src/App.js

```diff
@@ -27,23 +35,33 @@ function App() {
 
         <Route path='/'>
           <div className="App">
-            <VertNav navLinks={['intro', 'caesar', 'atbash', 'vigenere', 'recap', 'decoding', 'game-link', 'closing']} />
+            <VertNav navLinks={['intro', 'atbash', 'caesar', 'vigenere', 'recap', 'decoding', 'game-link', 'closing']} 
+              callbacks={{
+                atbash: () => { setScrollAtbash(true); }, 
+                caesar: () => { setScrollCaesar(true); }, 
+                vigenere: () => { setScrollVigenere(true); }, 
+                recap: () => { setScrollRecap(true); }, 
+                decoding: () => { setScrollDecoding(true); }, 
+                'game-link': () => { setScrollGame(true); }, 
+                closing: () => { setScrollClosing(true); }, 
```

Comment:
Hmm. Looking at this, things are rather verbose. I wonder if there's something I could do to on the `<VertNav />` end to make your life easier 🤔